.github/actions/set-python/action.yml

```diff
@@ -15,9 +15,9 @@ runs:
   steps:
     - name: Output python path
       id: python_run
-      shell: bash
+      shell: bash -l {0}
```

Comment:
What does this do?

Reply:
Seems to be required for using shells when using the conda action.
This ensures shell has conda path & the like setup appropriately (based on instructions from conda action).